plugin/adsk/plugin/ProxyShape.h

```diff
@@ -15,11 +15,12 @@
 //
 #pragma once
 
-#include "base/api.h"
+#include <mayaUsdPlugin/base/api.h>
 
+#include <mayaUsd/base/api.h>
```

Comment:
What requires this header? A quick search of the directory didn't show anything using `MAYAUSD_CORE_*`, `MAYAUSD_MACROS_*` or `MAYAUSD_TEMPLATE_*`.